providers/dns/exec/exec.go

```diff
@@ -0,0 +1,53 @@
+// Package exec implements a DNS provider for solving the DNS-01 challenge using
+// a user-provided executable.
+package exec
```

Comment:
Can the package be named something less "stdliby"? Like `dnsexec` maybe? Then we can ditch the renaming of the os/exec package.